### ISSUE TITLE
quizziz hack gone

### ISSUE THREAD

**Issue by codergautam** [EastArctica/JS-Plugins#18]
Question.structure.answers is not a thing anymore in api v4. rip quizizz hack

**Comment by EastArctica** [comment]
Yep, Question.structure.answers isn't a thing anymore in api v4. However there are still ways to get the answers with other api's, Please don't release them without an abstraction layer though! We don't want it patched haha. Close this if you don't have any further questions.
